r/R/dplyr-eval.R

```diff
@@ -156,6 +152,8 @@ arrow_mask <- function(.data, aggregation = FALSE) {
   # TODO: figure out what rlang::as_data_pronoun does/why we should use it
   # (because if we do we get `Error: Can't modify the data pronoun` in mutate())
   out$.data <- .data$selected_columns
+  # Add the aggregations list to collect any that get pulled out when evaluating
+  out$..aggregations <- empty_named_list()
```

Comment:
Any reason to have this be `..aggregations` instead of `.aggregations` like `.data`? Is it intended to be a signifier it's more internal than the `.data` is? 

It's no big deal either way, just a little funny looking now that it's here.

Reply:
Double secret ;) IDK, you're right, I can delete a dot.